tasks/main.yml

```diff
@@ -19,3 +19,9 @@
 
 - include_tasks: agent6-win.yml
   when: not datadog_agent5 and ansible_os_family == "Windows"
+
+- include_tasks: post_tasks/*.yml
```

Comment:
Should we ignore errors in post/pre tasks ?

We should also document this in the README

Reply:
Good point. I would say we shoudn't, since it wouldn't give users any hints as to where the issue(s) lie should they come across any while adding pre and post tasks?